src/main.js

```diff
@@ -1,44 +1,32 @@
 // Recomendation of babeljs (https://babeljs.io/docs/en/babel-polyfill)
-import 'core-js/stable'; // only stable feature also is possible with only `core-js`
-import 'regenerator-runtime/runtime'; // To ensure that regeneratorRuntime is defined globally
-import '@mdi/font/css/materialdesignicons.css'; // https://vuetifyjs.com/en/features/icon-fonts/#material-design-icons
-
-import './VueComposition';
-import Vue from 'vue';
-import VueTimeago from 'vue-timeago';
-import vueFilterPrettyBytes from 'vue-filter-pretty-bytes';
-import VueClosable from 'vue-closable';
-import App from './App';
-import vuetify from './plugins/vuetify';
-import router from './router';
-import store from './store';
+// Todo: see if core-js/regenerator-runtime are still necessary
+import "core-js/stable"; // only stable feature also is possible with only `core-js`
+import "regenerator-runtime/runtime"; // To ensure that regeneratorRuntime is defined globally
```

Comment:
Any idea perhaps what these requirements are used for/by?

Reply:
they have to do with backwards compatibility polyfilling. But I am not sure if they are still necessary, using the new vite setup, this is why I added the todo. You could argue we should figure this out for this migration.